web/themes/custom/camp/js/camp.main.js

```diff
@@ -101,6 +101,7 @@
        * @param {object} target Dom element.
        */
       function stickyHeader(wrapper, target) {
+        return;
```

Comment:
ignorer denne. Det er for at det hele ikke crasher. Jeg fikser sticky header menu i nyt PR senere.